content/news/20210123-landscape-now.md

```diff
@@ -0,0 +1,72 @@
++++
+title = 'Landscape, Now'
+description = 'Landscape does exactly what it appears to do and nothing else. There are no ads or tracking, there are no gimmicks to try to keep you engaged or addicted, and there’s no quiet uncertainty that Landscape might disappear or change drastically.'
+date = 2019-11-27
```

Comment:
This date appears to be Ancient? Is this that old?

Reply:
Oh yeah, jfc